enforcer-parent/commons/src/test/java/org/wso2/choreo/connect/enforcer/commons/model/RequestContextTest.java

```diff
@@ -101,6 +105,9 @@ private void testPathParamValues(String rawPath, String basePath, String pathTem
         builder.matchedAPI(new APIConfig.Builder("Petstore").basePath(basePath).build());
         builder.pathTemplate(pathTemplate);
         RequestContext requestContext = builder.build();
+        if (pathParamName == null) {
+            return;
+        }
```

Comment:
i dont think its safe to have this. shall we remove this as we are not having a null case in our tests?
```suggestion
```